src/Solana/Program.cpp

```diff
@@ -62,7 +62,7 @@ Address TokenProgram::defaultTokenAddress(const Address& mainAddress, const Addr
  * Based on solana code, find_program_address()
  * https://github.com/solana-labs/solana/blob/master/sdk/program/src/pubkey.rs#L193
  */
-Address TokenProgram::findProgramAddress(const std::vector<TW::Data>& seeds, const Address& programId) {
+Address TokenProgram::findProgramAddress(const std::vector<TW::Data>& seeds, [[maybe_unused]] const Address& programId) {
```

Comment:
hmm double check and remove it?

Reply:
This is a bug, parameter is not used but a constant, but currently is always invoked with the same constant.  I will fix this.

Reply:
Handled in #2406